pkg/controllers/provisioning/suite_test.go

```diff
@@ -220,6 +220,43 @@ var _ = Describe("Provisioning", func() {
 		Expect(len(nodes.Items)).To(Equal(1))
 		ExpectScheduled(ctx, env.Client, pod)
 	})
+	It("should expect terminationGracePeriod to be nil when nodepool and global value is not set", func() {
```

Comment:
It doesn't look like this description is right, is it? We're setting the global to 98h in this test.

Reply:
Updated the description